frontend/src/utilities/js-message-handler-binding.ts

```diff
@@ -1,4 +1,4 @@
 // This file is instantiated by wasm-bindgen in `/frontend/wasm/src/lib.rs` and re-exports the `handleResponse` function to
 // provide access to the global copy of `response-handler.ts` with its shared state, not an isolated duplicate with empty state
 
-export { handleResponse } from "@/utilities/response-handler";
+export { handleResponse } from "@/utilities/js-message-dispatcher";
```

Comment:
This file should be called `js-message-dispatcher-binding.ts` (or maybe `js-message-dispatcher-reexport.ts` if you think that's more clear) since it's meant to shadow the name of the "real" file (the reexported one, `js-message-dispatcher.ts`).